include/deal.II/numerics/vector_tools.templates.h

```diff
@@ -6724,11 +6724,8 @@ namespace VectorTools
                const InVector        &fe_function,
                const Point<spacedim>      &point)
   {
-#ifdef DEBUG
-    const FiniteElement<dim> &fe = dof.get_fe();
-    Assert(fe.n_components() == 1,
+    Assert(dof.get_fe().n_components() == 1,
            ExcMessage ("Finite element is not scalar as is necessary for this function"));
-#endif
```

Comment:
I don't disagree with the bikeshedding comment. It's just that if you don't fix things that could be cleaned up right away, it never happens. It's not a comment on anyone else's changes. It's just regular maintenance.